t/full-stack.t

```diff
@@ -103,31 +105,28 @@ sub check_scheduled_job_and_wait_for_free_worker ($worker_class) {
         $has_relevant_job = 1;
         last;
     }
-    ok $has_relevant_job, "job with worker class $worker_class scheduled"
-      or diag explain 'scheduled jobs: ', \@scheduled_jobs;
+    Test::More::ok $has_relevant_job, "job with worker class $worker_class scheduled"
+      or Test::More::diag explain 'scheduled jobs: ', \@scheduled_jobs;
 
     # wait until there's not only a free worker but also one with matching worker properties
     # note: Populating the database is not done atomically so a worker might already show up but relevant
     #       properties (most importantly WEBSOCKET_API_VERSION and WORKER_CLASS) have not been populated yet.
     my ($elapsed, $free_workers) = (0, []);
-    for (; $elapsed <= $setup_timeout; $elapsed += sleep 0.1) {
+    for (; $elapsed <= $setup_timeout; $elapsed += sleep $setup_poll_interval) {
```

Comment:
Hrmmm why is this `not covered by tests`? That seems odd. We should have full coverage here? Maybe the variable is the problem?

Reply:
I added two diag messages.
This is what I got on CircleCI:
![fullstack-coverage-circleci](https://user-images.githubusercontent.com/688850/154317935-ce77cc00-f9a6-4b88-98c6-975428024a00.png)
Test output: https://app.circleci.com/pipelines/github/perlpunk/openQA/258/workflows/06822a3c-9ffb-422f-ab28-9bd425e78080/jobs/1460?invite=true#step-110-128
```
# ================ 0 18 0.1
    # ==== 0
    # ==== 0.100072
    # ==== 0.200194
    # ==== 0.300301
    # ==== 0.400373
    # ==== 0.500481
    # ==== 0.600602
    # ==== 0.700674
```
So the diag lines were both executed.

and this is local:
![fullstack-coverage-local](https://user-images.githubusercontent.com/688850/154317938-379cb5f1-63b1-4edc-9cca-21b517d1bded.png)
with the only difference of 6 instead of 18:
```
# ================ 0 6 0.1                                                                                                  
# ==== 0                                                                                                                    
# ==== 0.100092                                                                                                             
# ==== 0.20019                                                                                                              
# ==== 0.300283                                                                                                             
# ==== 0.400376                                            
# ==== 0.500496                                            
```

I can't imagine what goes wrong with Devel::Cover here.